region/lowernorfair/east/Fast Pillars Setup Room.json

```diff
@@ -2370,11 +2510,35 @@
       "requires": [
         "h_heatProof",
         {"resetRoom": {"nodes": [2, 3]}},
-        {"refill": ["PowerBomb"]}
+        {"cycleFrames": 200}
       ],
       "resetsObstacles": ["A"],
```

Comment:
By laying a Power Bomb before collecting the last Power Bomb drop, we could end with the obstacle cleared while still having full Power Bombs. So this could be `"clearsObstacles": ["A"]` instead of `"resetsObstacles": ["A"]`. I guess it would need a separate strat though, since we're not necessarily assuming Morph is available here (since Violas can also be farmed for energy). Not sure if it's worth adding, just wanted to mention :shrug: